README.md

```diff
@@ -1,70 +1,142 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
 
-## Available Scripts
+  <h3><b>Hello React Front-End</b></h3>
 
-In the project directory, you can run:
+</div>
 
-### `npm start`
+# 📗 Table of Contents
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)   
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)    
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+# 📖 [Hello React Front-End] <a name="about-project"></a>
 
-### `npm test`
+> This is a web application front-end built using React JS that uses an API web service built with ruby on rails.
 
```

Comment:
![image](https://github.com/lenkon/hello-react-front-end/assets/93039359/a5610e00-18ea-4492-85ca-09a764d28bdf)

[OPTIONAL BUT HIGHLY RECOMMENDED]

- [ ] It would be nice if you merge both lines to get a good summary because i see you have two lines of summary at different place 😨 Kindly merge them to make your readme more descriptive and professional 👍

Reply:
Hi @HAMMAS-SALEEM ,

Thanks for the suggestion. I have removed the second header so that it would have a single summary.

Best Regards,
K. Inoulen Shanongba